playbooks/configure-release.yml

```diff
@@ -120,16 +120,25 @@
         force: "{{ osa_force_clone | default(false) }}"
 
     - name: Copy basic files into place
```

Comment:
This task, the "Sync configuration for RPC-OpenStack files" task seems like it's doing the same thing twice. The initial intent of these tasks was to create the base directory if needed using the example files from OSA and then sync our files as needed. If we're just going to plain sync we should create the base dir and rsync just the one time.

Reply:
Well spotted, that's been corrected to copy from the correct source.